app/components/settings/categories/SupportSettings.scss

```diff
@@ -15,49 +15,41 @@
   }
 
   a,
-  button,
   p {
     color: var(--theme-support-settings-text-color);
     font-family: var(--font-regular);
     font-size: 14px;
     line-height: 19px;
   }
 
-  a,
-  button {
+  a {
     color: var(--theme-support-settings-link-color);
   }
-
-  button {
-    cursor: pointer;
-    letter-spacing: 1px;
-  }
-
+  
   p {
     margin-bottom: 10px;
   }
-}
+  
+  .link {
```

Comment:
maybe link should be more of a global style 🤔

Reply:
@nicarq 
Yes, exactly.
As there are many place which we could fix.
Have listed out places over here: https://app.clubhouse.io/emurgo/story/1709/change-underline-to-custom-underline

So thinking to fix all places(including this), in a separate PR.
Will that work ?

Reply:
yes, that’s good!